interfaces/builtin/desktop.go

```diff
@@ -463,12 +463,25 @@ dbus (receive, send)
     interface=org.freedesktop.DBus.Properties
     peer=(label=unconfined),
 
-# Allow access to the regular xdg-desktop-portal APIs
+# Allow access and introspection to the regular xdg-desktop-portal APIs
```

Comment:
The new rules here don't seem to add introspection. If that's needed, shouldn't there be some rule relating to `org.freedesktop.DBus.Introspectable`?

Reply:
Yes, I meant the properties introspection here. But indeed we could widen it to full introspection.

Reply:
done